ui/apps/platform/src/Components/CompoundSearchFilter/utils/utils.tsx

```diff
@@ -80,16 +80,6 @@ export function getDefaultAttributeName(
     return attributes?.[0]?.displayName;
 }
 
-export function ensureStringArray(value: unknown): string[] {
-    if (Array.isArray(value) && value.every((element) => typeof element === 'string')) {
-        return value;
-    }
-    if (value === 'string') {
-        return [value];
-    }
-    return [];
-}
```

Comment:
Pulling this into the shared `ensure.ts` file because it's used in multiple places now

Reply:
I feel like there was another implementation of this somewhere in VM, but I can't seem to find it off hand. 🤔 

If I happen to see it I'll refactor to use the common util.

Reply:
Let me know. I couldn't find any from a quick search. I wonder if you're thinking of the `CompoundSearchFilter`. I had initially started making those util functions under that directory. I've been slowly extracting those out as necessary